posts/2021-01-26-ee9-messaging-security-21002-beta.adoc

```diff
@@ -116,6 +116,35 @@ The following features can be included in your server.xml:
 
 Additional information about the Jakarta Security 2.0 specification can be found link:https://jakarta.ee/specifications/security/2.0/[here].
 
+[#xmlws]
+=== Jakarta XML Web Services 3.0
+
+Jakarta XML Web Services 3.0 is an API that enables applications to create SOAP based Web Services. Support provides the following updated APIs: XML Web Services 3.0, Jakarta Web Services Metadata 3.0, Jakarta SOAP with Attachments 2.0. Support also introduces the `xmlWS-3.0` feature, as well as the Application Client Container support for XML Web Services 3.0 via the `jakartaeeClient-9.0` convenience feature.
+
+The `xmlWS-3.0` feature provides the Jakarta EE 9 XML Web Services 3.0 tools for Top Down and Bottom Up Web Services Creation: `wsgen` and `wsimport`. These can be found in a new location in the Open Liberty image: wlp/bin/xmlWS/. Both commands preform nearly identical to the Jakarta EE 8 tooling, with the exception that the `-target` argument is no longer required.
+
+The implementation of Jakarta XML Web Services 3.0 in Open Liberty is based on Apache CXF 3.4.1. This provides numerous improvements and fixes over the forked version of CXF 2.6.2 in use by the Jakarta EE 8 `jaxws-2.3` feature (Note: Not all CXF 3.4.1 functionality is available in `xmlWS-3.0`, but watch this space for updates.)
```

Comment:
I think maybe this paragraph should be removed... I don't like advertising a specific implementation version or hinting at support for future function.

Reply:
Removed paragraph and changed `preform` -> `perform`